lab-jaren/test/game-router-test.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+const expect = require('chai').expect;
+const Game = require('../model/game.js');
+const storage = require('../lib/storage.js');
+const superagent = require('superagent');
+
+const apiURL = `http://localhost:${process.env.PORT || 3000}`;
+require('../server.js');
+
+describe('testing /api/games', function() {
+  describe('testing POST', function() {
+    describe('with valid input', function() {
+      it('should return a game', done => {
+        superagent.post(`${apiURL}/api/games`)
+        .send({
```

Comment:
You can save yourself a lot of trouble and refactor your first POST test.

At the top of your entire describe block, you can set your mock data at line 10:
```
let example = {
  title: 'XCOM 2',
  genre: 'strategy/tactics',
  developer: '2K Games, Inc.',
  publisher: 'Firaxis Games, Inc.',
  platforms: 'Windows, Linux, Mac, PS4, Xbox One',
  ratingESRB: 'Teen',
  releaseDate: 'Feb 05, 2016',
};
```
Then your test can be refactored like so:
```
    describe('with valid input', function() {
      it('should return a game', done => {
        superagent.post(`${apiURL}/api/games`)
        .send(example)
        .then(res => {
          console.log('res.body', res.body);
          console.log('example: ', example);
          for(let key in example){
            expect(example[key]).to.equal(res.body[key]);
          }
          expect(Boolean(res.body.id)).to.equal(true);
          done();
        })
        .catch(done);
      });
    });
```

I pulled down your lab and refactored in this way and it worked. Let me know if you have questions.